stable/database/templates/statefulset.yaml

```diff
@@ -63,10 +63,22 @@ spec:
       - name: init-disk
         image: {{ template "init.image" . }}
         imagePullPolicy: {{ default "" .Values.busybox.image.pullPolicy | quote }}
-        command: ['chmod' , '770', '/var/opt/nuodb/archive', '/var/log/nuodb']
+        command:
+          - 'chmod'
+          - '770'
+          - '/var/opt/nuodb/archive'
+          - 'var/opt/nuodb/backup'
```

Comment:
this does not belong here!

Reply:
Wherever it lands, there are a few points on the change:
1. the new code seems to no longer chmod on /var/log/nuodb
    1a and this chmod needs to be 'chmod -R'
2. the path 'var/opt/nuodb/backup' should be '/var/opt/nuodb/backup'